src/plugins/dam/components/document-asset-picker-modal/delegate.js

```diff
@@ -0,0 +1,24 @@
+import throttle from 'lodash/throttle';
+import searchNodes from '@triniti/cms/plugins/ncr/actions/searchNodes';
+import { pbjxChannelNames } from '@triniti/cms/plugins/dam/constants';
+import clearChannel from '../../../pbjx/actions/clearChannel';
+import schemas from './schemas';
+
+export default (dispatch, { searchDocumentsQ }) => ({
```

Comment:
@arunkarnati yeah its wonky i agree. i twitch a bit either way - including the validation in a component that doesnt use it, or not having it from an ownProps key. tbh i dont know what kind of value we even get from props validation. like, it may have helped me catch something faster during development but im not convinced it actually prevents bugs or improves code quality :/

Reply:
@efjacobson - I like to look at the component's props declaration as a component's quick definition before I use it in a parent component. I don't have strong feelings either way.